### DISCUSSION TITLE
Blender and LibreOffice, XFCE and VNC

### DISCUSSION THREAD

**Discussion by anonimo82** [nix-community/nix-on-droid#256] [Q&A]
Hello everyone.
I need to install Blender 2.79b and LibreOffice 7.2.7.2, but they're not available with `nix-env -qa`.
Is it there a workaround?

Also, I'd like to install XFCE and VNC Server to login in X/Wayland, how can I do?

Thanks for any help, regards.
Have a nice day =)

**Comment by t184256** [comment]
> I need to install Blender 2.79b and LibreOffice 7.2.7.

That's eerily specific =) You'd have to find nixpkgs version which has them or package them yourself.

Mind that if you need to not just install them, but also use them, you'd be in trouble. There'll be no GPU acceleration and the performance will be abysmal. I don't recommend thatpursuing that.

> Also, I'd like to install XFCE and VNC Server to login in X/Wayland, how can I do?

You'd probably be much better served by Termux distribution: higher performance (no proot), more support for running graphical apps, more conventional package management.
